trulens_eval/examples/quickstart/langchain_retrieval_multi_agents.ipynb

```diff
@@ -0,0 +1,348 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    ! pip install trulens_eval==0.15.3 langchain&gt;=0.0.263 unstructured&gt;=0.0.1 chromadb&gt;=0.0.1
</pre><p>Update with actual langchain version used. Try not to use &gt;= . If need to use &gt;=, put quotes around requirements as some shells will interpret as a piping.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/truera/trulens/pull/500/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
updated